043-SQLModernization/Student/Challenge03.md

```diff
@@ -0,0 +1,58 @@
+# Challenge 3 - Performance & Tuning
+
+[< Previous Challenge](./Challenge02.md) - **[Home](../README.md)** - [Next Challenge>](./Challenge04.md)
+
+## Notes & Guidance
+
+Although SQL Server is easier than ever to deploy with Azure SQL Database, containers, and VMs, knowing how to properly use the monitoring and evaluation tools, like Query Store, can make a difference between an application that simply "performs ok" to one that "flies."  While SQL Server has evolved with features over the years, it's a good idea to understand how these changes might impact an application.
+
+## Description
+
+The purpose of this challenge is threefold:
+* Explore new features of SQL Server that may improve performance intrinsically.
+* Ensure would-be data engineers and DBAs are comfortable monitoring, evaluating performance. 
+* Leverage newer tools like Azure Data Studio Notebooks and [Azure Monitor SQL Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/sql-insights-overview).
+
+### Explore new features
+
+To begin, download and open the [Intelligent Query Processing notebook](./resources/SQLWTH_Challenge3_IntelligentQueryProcessing.ipynb) using Azure Data Studio or another tool of your choice that can work with a standard Jupyter Notebook. The Notebook will walk you through the test, comparing performance of an analytical query using compatibility mode 130 vs 150.
```

Comment:
I'm not sure the expected behavior for linking to a Jupiter Notebook.  I believe you should be using the raw link for this? When I click on it now, it open up the repo in GH which is a no-no for WTH student guides.

Reply:
done, will be in PR